src/routes.c

```diff
@@ -3704,6 +3706,8 @@ WOLFSENTRY_API wolfsentry_errcode_t wolfsentry_route_table_iterate_end(
     WOLFSENTRY_RETURN_OK;
 }
 
+#ifndef WOLFSENTRY_NO_STDIO_STREAMS
```

Comment:
`hexdigit_ntoa()` is a replacement for stdio logic, and `wolfsentry_route_format_address()` is an API routine and doesn't have stdio deps -- always build them.